Dockerfile

```diff
@@ -1,23 +1,25 @@
-FROM node:14-alpine
-
-#Setup folders
-RUN mkdir -p /home/node/app/node_modules && chown -R node:node /home/node/app
-WORKDIR /home/node/app
+# ---- Build ----
+FROM node:14-alpine AS build
 
-#Install node packages
+WORKDIR /usr/src/app
 COPY package*.json ./
-RUN npm install
-
-#Copies source
-COPY . .
-COPY --chown=node:node . .
 
-USER node
+# --ignore-scripts to prevent prisma generate from running.
+# No use until we got the source code
+RUN npm install --ignore-script
 
-EXPOSE 8080
+COPY . .
+RUN npx prisma generate
+RUN npm run build
 
-#Set environment varibales
+# ---- Release ----
+FROM node:14-alpine
 
+WORKDIR /usr/src/app
+COPY --from=build /usr/src/app/dist ./dist
```

Comment:
Using multi build and skipping unnecessary content takes the image from 597MB to 376MB

Reply:
On GCP it's a bit different, but gets smaller nonetheless 
![Screenshot from 2023-05-29 13-49-43](https://github.com/stiftelsen-effekt/effekt-backend/assets/3483136/c8e0967c-d58c-4669-9df7-e6ff2dc4b601)